jre_emul/jre_sources.mk

```diff
@@ -1075,9 +1076,13 @@ JAVA_PUBLIC_SOURCES_FILE = \
   java/nio/file/FileSystemException.java \
   java/nio/file/FileSystemLoopException.java \
   java/nio/file/FileSystemNotFoundException.java \
+  java/nio/file/FileSystems.java \
+  java/nio/file/FileTreeIterator.java \
```

Comment:
Should be in JAVA_PRIVATE_SOURCES_FILE.

Reply:
Does this refer to both sources or only FileTreeIterator?

Reply:
Just FileTreeIterator, as I made a separate comment for each package-private class. Basically, all package-private classes go in PRIVATE, all public and protected in PUBLIC.